### DISCUSSION TITLE
No wagmi client found?

### DISCUSSION THREAD

**Discussion by salieflewis** [wevm/wagmi#643] [Q&A]
I am trying to setup connect wallet functionality like in the examples. My app is wrapped in a <WagmiConfig> Provider.

Not really sure why I am getting this error. Here is what my code looks like.

```
const client = createClient({
  autoConnect: true,
  connectors: [new MetaMaskConnector({ chains })],
  provider,
});

const App = () => {
  return (
    <WagmiConfig client={client}>
      <Connect />
      <Approve />
    </WagmiConfig>
  );
};
```

**Comment by tmm** [comment]
Can you put together a CodeSandbox or GitHub repo reproducing the issue?

**Reply by salieflewis** [reply]
Will do.
